### PR TITLE
fix(curriculum): update video id, review and quiz for basic html module

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This PR replaces the id for 2 lecture videos. Another 2 videos are still missing from this section because they are not ready.

References to inline and block-level elements have been removed since they are taught in the basic CSS module

### CODE REVIEW

**Comment by ilenia-magoni** [comment]
Which of the following is the correct minimalist boilerplate?

--answer--
```html
<!DOCTYPE html>
<html>
<head>
</head>
<body>
</body>
</html>
```

--distractors--
```html
<html>
<head>
<body>
<!DOCTYPE html>
</body>
</head>
</html>
```

```html
<head>
<body>
<!DOCTYPE html>
<html>
</html>
</body>
</head>
```

```html
<!DOCTYPE html>
<body>
<head>
<html>
</html>
</head>
</body>
```

**Comment by ilenia-magoni** [comment]
Which of the following is a correct value for the id attribute?

--answer--
`id="main-content"`

--distractors--
`id="123content"`
`id="main content"`
`id="main@content"`

**Comment by ilenia-magoni** [comment]
Which of the following is the correct syntax for a `img` element?

--answer--
`Description of the image`

--distractors--
``

`</img>`

`Description of the image</img>`

**Review by majestic-owl448 — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/review-basic-html/67072fc183c7ca6c588feb4d.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ Review the concepts below to prepare for the upcoming quiz.
 ## HTML Basics
 
 - **Role of HTML**: Foundation of web structure.
-- **Block-level elements**: The `div` element is a generic HTML element that does not hold any semantic meaning. It is used as a generic container to hold over HTML elements.
+- **HTML Elements**: Used to represent content on the page. Most of them are made by an opening and a closing tag (e.g., `<h1></h1>`, `<p></p>`).
+- **Div elements**: The `div` element is a generic HTML element that does not hold any semantic meaning. It is used as a generic container to hold over HTML elements.
```
Comment (jdwilkin4) [inline curriculum/challenges/english/25-front-end-development/review-basic-html/67072fc183c7ca6c588feb4d.md:16]: small thing, but I think div should be all lowercase here

```suggestion
- **`div` elements**: The `div` element is a generic HTML element that does not hold any semantic meaning. It is used as a generic container to hold over HTML elements.
```

**Review by jdwilkin4 — APPROVED** [review]
Made one small suggestion but otherwise good 👍

**Pull request merged**